assets/app/views/about.html

```diff
@@ -0,0 +1,65 @@
+<div>
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <div class="about">
+          <div class="row">
+            <div class="col-md-2 about-icon gutter-top hidden-sm hidden-xs">
+              <span class="pficon pficon-openshift" aria-hidden="true"></span>
+            </div>
+            <div class="col-md-9">
+              <div class="page-header page-header-bleed-right">
+                <h1>OpenShift by Red Hat&reg;</h1>
+              </div>
+
+              <h2>About</h2>
+              <p><a href="https://openshift.com">OpenShift</a> is Red Hat's Platform-as-a-Service (PaaS) that allows developers to quickly develop, host, and scale applications in a cloud environment.</p>
```

Comment:
I wonder, if this is the about page for Origin, then we may want to have the link go to openshift.org
Likewise, at some point we may want to specific links to the offering page for Enterprise and Dedicated?.

Though I think for an initial pass, it would be fine just defaulting to openshift.com as it has redirects already